src/Orc.FilterBuilder/Orc.FilterBuilder.Shared/Expressions/PropertyExpression.cs

```diff
@@ -38,55 +38,55 @@ private void OnPropertyChanged()
 
             if (Property.Type.IsEnumEx())
             {
-                CreateDataTypeExpressionIfNotCompatible(() => new StringExpression());
+                DataTypeExpression = new StringExpression();
```

Comment:
Yeah, my bad, I didn't check tests as there was only 2 or 3 tests so I assumed it not worth checking. I will fix it, sorry.